### PR TITLE
aws-vault: add page

### PR DESCRIPTION
**Pull request by anaynayak**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

File: pages/common/aws-vault.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+> A vault for securely storing and accessing AWS credentials in development environments.
+> More information: <https://github.com/99designs/aws-vault>.
+
+- Add credentials to the secure keystore:
+
+`aws-vault add {{profile}}`
+
+- Execute a command with AWS credentials in the environment:
+
+`aws-vault exec {{profile}} -- aws s3 ls`
```
Comment (owenvoke) [inline pages/common/aws-vault.md:12]:
```suggestion
`aws-vault exec {{profile}} -- {{aws s3 ls}}`
```

**Review by owenvoke — CHANGES_REQUESTED** [review]
LGTM! Just one minor change to add additional token syntax.

File: pages/common/aws-vault.md (hunk 1 above)
Comment (anaynayak) [reply pages/common/aws-vault.md]: thanks. fixed.

**Review by owenvoke — APPROVED** [review]
LGTM, thank you!

**Pull request merged**